File: app/src/main/java/com/dabutu/gympet/Routine/Routine.java
package com.dabutu.gympet.Routine;

public class Routine {
    private String name;
    private int numberOfExercises;

    public Routine(String name, int numberOfExercises) {
        this.name = name;
        this.numberOfExercises = numberOfExercises;
    }

    public String getName() {
        return name;
    }

    public int getNumberOfExercises() {
        return numberOfExercises;
    }
}

File: app/src/main/java/com/dabutu/gympet/Routine/Routine.kt
package com.dabutu.gympet.Routine

import java.io.Serializable

data class Routine(
    val title: String = "",
    val exerciseCount: Int = 0,
    var exercises: List<Exercise> = listOf()
) : Serializable {
    constructor() : this("", 0, listOf())
}
